renku/core/workflow/plan.py

```diff
@@ -644,3 +684,90 @@ def get_composite_plans_by_child(plan: AbstractPlan, plan_gateway: IPlanGateway)
     composites_containing_child = [c for c in composites if {p.id for p in c.plans}.intersection(derivatives)]
 
     return composites_containing_child
+
+
+@inject.autoparams("activity_gateway", "plan_gateway")
+def get_active_plans(activity_gateway: IActivityGateway, plan_gateway: IPlanGateway) -> List[AbstractPlan]:
+    """Get all plans that are active in the project.
+
+    Active means not deleted and with at least one of the used/created files existing.
```

Comment:
I don't see that we filter the results based on the `touches_existing_files`; we add all the plans regardless. If that's intended then this description should change and wise-versa.

Reply:
Right, we decided to return everything and have the UI filter based on `touches_existing_files`, I have to rename/reword that.